api/v2/helpers/bouquets/bouquets_helper.py

```diff
@@ -9,3 +9,15 @@ def query_all_bouquets():
         bouquets = BouquetsSchema(many=True).dump(all_bouquets)
 
     return bouquets
+
+
+def create_bouquet(request_data):
```

Comment:
This function is too brief, specific and concerns mostly the model. It does not deserve to be a helper function.